src/olympia/devhub/templates/devhub/new-landing/components/blog_posts.html

```diff
@@ -0,0 +1,24 @@
+<section class="DevHub-BlogPosts">
+  <h2>
+    <a title="{{ _('Subscribe to this feed') }}" href="http://blog.mozilla.com/addons/feed/">
+      {{ _('Latest News') }}
+    </a>
+  </h2>
+  <ul>
+    {% for post in blog_posts %}
+    <li>
```

Comment:
I always like the look of these when they're indented, source code readability matters more to me than rendered HTML looking pretty (which it won't anyway).